ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -6087,18 +6251,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-nightly-x86-data-path-6nodes
-  reporter_config:
```

Comment:
@SachinNinganure For all cronjobs you added or updated, please add or keep the slack reporter.

Reply:
done!!

Reply:
for the below test, QE team don't have the plan to add cron job for non-ipsec for Azure now. If you want to keep it for comparison work or if PerfScale Engineering team wants to keep it, we can keep it. If not, please make it as - always_run: false instead of a cronjob

will check with @krishvoor